lean4-infoview/src/infoview/info.tsx

```diff
@@ -273,8 +274,15 @@ function InfoAux(props: InfoProps) {
 
         function onError(err: any) {
             const errS = typeof err === 'string' ? err : JSON.stringify(err);
-            setError(`Error fetching goals: ${errS}`);
-            setStatus('error');
+            // we need to check if this value is empty or not, because maybe we are assigning
+            // a message error with an empty error
+            if (errS === '{}' || errS === undefined) {
+                setError(undefined);
+            }
+            else {
```

Comment:
```suggestion
```

We can now revert this part since we call `setError(undefined)` directly, right?

Reply:
Not really, there is an issue here. When the server crashes and then is restarted the variable 'errS' stays in a weird state and it means that after we restart the server, we can't see the 'No info found' message in the infoview (when we put the cursor in a blank space). That's why we need to check if 'errS' is {} or undefined and set the error as 'undefined'

![image](https://user-images.githubusercontent.com/40707114/170582840-4b87ce01-1461-410d-b63c-c2ca7c93fb5d.png)

Reply:
Hmm, this is weird, but at least we show the right error message now..